### DISCUSSION TITLE
Benchmark (Simple Product w/Product Options) versus (Configurable Product)

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#1690] [Questions & Answers]
For a long time only simple products with custom options such as size and color were used in the stores I provide maintenance for. Being about stores that had all the variants in stock there was no problem of creating dependencies between the variants. 

Recently I encountered a new situation where not all variants are available in stock and then I had to analyze between going for (Configurable Product + Aitoc Configurable Product Pro) or (Simple Product + Mageworx Advanced Product Options). If I choose for the first option, I don't really like the fact that for a product that has 6 sizes and 6 colors I will get 36 simple products. In addition, obviously if there are 1.000 products in the first variant there will be 36.000 simple products while in the second variant only 1000.

Has anyone ever made a benchmark of a store with simple products and one with configurable?

**Comment by brianroyce** [comment]
We did not do any benchmarking we should have but we did build something we call a “Manufactured Product Type” we use raw goods to create products — Kayla will provide an example.



On Jun 18, 2021, at 11:28 AM, ADDISON ***@***.******@***.***>> wrote:



For a long time only simple products with custom options such as size and color were used in the stores I provide maintenance for. Being about stores that had all the variants in stock there was no problem of creating dependencies between the variants.

Recently I encountered a new situation where not all variants are available in stock and then I had to analyze between going for (Configurable Product + Aitoc Configurable Product Pro) or (Simple Product + Mageworx Advanced Product Options). If I choose for the first option, I don't really like the fact that for a product that has 6 sizes and 6 colors I will get 36 simple products. In addition, obviously if there are 1.000 products in the first variant there will be 36.000 simple products while in the second variant only 1000.

Has anyone ever made a benchmark of a store with simple products and one with configurable?
